monkey/infection_monkey/exploit/sshexec.py

```diff
@@ -188,12 +214,23 @@ def _exploit_host(self) -> ExploiterResultData:
                     self.exploit_result.error_message = f"SSH Skipping unknown os: {uname_os}"
 
                     if not uname_os:
+                        self.publish_propagation_event(
+                            target=self.host.ip_addr,
+                            propagation_success=False,
+                            error_message=self.exploit_result.error_message,
+                        )
+
                         logger.error(self.exploit_result.error_message)
-                    return self.exploit_result
+                        return self.exploit_result
```

Comment:
?

Reply:
The formatting went wrong. I will fix it.